packages/create-river-app/src/index.ts

```diff
@@ -0,0 +1,99 @@
+import * as spawn from 'cross-spawn'
+import minimist from 'minimist'
+import path from 'node:path'
+import fs from 'node:fs'
+import picocolors from 'picocolors'
+
+const getPackageManager = () => {
+    if (process.env.npm_config_user_agent) {
+        const agent = process.env.npm_config_user_agent
+        if (agent.startsWith('yarn')) return 'yarn'
+        if (agent.startsWith('npm')) return 'npm'
+        if (agent.startsWith('pnpm')) return 'pnpm'
+        if (agent.startsWith('bun')) return 'bun'
+    }
+    return 'npm' // Default to npm if no user agent is found
+}
+
+const argv = minimist(process.argv.slice(2))
+
```

Comment:
request - find a good create app script that takes arguments, copy their architecture and update this script to 1) accept --help, 2) move this create react-app logic into own file so that it's easy to add another command in the future.